test/integration/archive_install/spec.rb

```diff
@@ -0,0 +1,48 @@
+windows = os.windows?
+
+control 'archive_install' do
```

Comment:
Minor, these control blocks are optional and are more intended for when the inspec test is covering an official compliance check of some sort.  Since these are just custom tests I don't think we need them.